src/screenfull.js

```diff
@@ -99,10 +104,22 @@
 			}
 		},
 		onchange: function (callback) {
-			document.addEventListener(fn.fullscreenchange, callback, false);
+			this.on('change', callback);
 		},
 		onerror: function (callback) {
-			document.addEventListener(fn.fullscreenerror, callback, false);
+			this.on('error', callback);
+		},
+		on: function (event, callback) {
+			var evName = eventNameMap[event];
+			if (evName) {
+				document.addEventListener(evName, callback, false);
+			}
+		},
+		off: function (event, callback) {
+			var evName = eventNameMap[event];
+			if (evName) {
+				document.off(evName, callback, false);
```

Comment:
Uh, what's this? `document` doesn't have an `off` method. Does this run for you?